### PR TITLE
<wip> Push scylla-bench docker image using gocql version from the PR

### PR DESCRIPTION
**Pull request by sylwiaszunejko**

### CODE REVIEW

**Comment by dkropachev** [comment]
@sylwiaszunejko , please refere issue that it is related to in PR description

**Comment by sylwiaszunejko** [comment]
its just wip and testing stuff no need for the review right now

File: .github/workflows/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM golang:1.18
+
+ARG version
+ARG fork
+ENV version=${version}
+ENV fork=${fork:-scylladb/scylla-bench}
+
+RUN apt-get update; apt-get -y install unzip curl
```
Comment (mykaul) [inline .github/workflows/Dockerfile:8]:
```suggestion
RUN apt-get update && apt-get -y install --no-install-recommends unzip curl && apt-get clean &&  rm -rf /var/lib/apt/lists/*
```

File: .github/workflows/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+RUN curl -Lo sb.zip https://github.com/${fork}/archive/refs/${version}.zip ;\
```
Comment (mykaul) [inline .github/workflows/Dockerfile:10]: You really should:
1. Use && - there's no point in continuing commands if the former failed
2. Cleanup the zip file at the end.

(we'd like to keep this container small)

File: .github/workflows/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
+FROM golang:1.18
```
[the remark this answers is not in the document]
Comment (sylwiaszunejko) [reply .github/workflows/Dockerfile:1]: my mistake, I copied this Dockerfile https://github.com/scylladb/scylla-cluster-tests/blob/1e2ab0ec93d0975f75ac22d7197210be42a13a19/docker/scylla-bench/Dockerfile

[the remark this answers is not in the document]
Comment (mykaul) [reply .github/workflows/Dockerfile:1]: Well, it should be fixed too someday...

**Pull request closed without merging**